ast/ast.go

```diff
@@ -443,7 +453,8 @@ type Select struct {
 	Select token.Pos // position of "select" keyword
 
 	Distinct bool
-	AsStruct bool
+	AsStruct bool         // deprecated, use As
```

Comment:
I'm not sure this deprecation is necessary. Could we remove it simply?

Reply:
Removed https://github.com/cloudspannerecosystem/memefish/pull/122/commits/ff14fe6ee2485c523010146f06cc6ff60b8d37a8

It is because I didn't sure the deprecation policy of memefish.
Current policy: We can change any AST node without breaking change notification because there are no release version.
Is it correct?

Reply:
I have no time to maintain codes with some deprecations. Also, we have no plan to publish a major version, so we can easily introduce breaking changes IMO.

Reply:
Thank you for answering!

We can make breaking changes to v0.x.y at any time, but the release process itself is a significant effort, so I think the current policy is fine.